pkg/apis/nats/v1alpha2/cluster.go

```diff
@@ -126,6 +130,15 @@ type ClusterSpec struct {
 	ExtraRoutes []*ExtraRoute `json:"extraRoutes,omitempty"`
 }
 
+// ServerConfig is extra configuration for the NATS server.
+type ServerConfig struct {
+	Debug          bool   `json:"debug,omitempty"`
+	Trace          bool   `json:"trace,omitempty"`
+	WriteDeadline  string `json:"write_deadline,omitempty"`
+	MaxConnections int    `json:"max_connections,omitempty"`
```

Comment:
While at it, maybe add MaxSubscriptions, MaxControlLine, LogTime?  Wdyt?

Reply:
sgtm, will too

Reply:
Added more extra fields for the limits, thanks for taking a look.